### PR TITLE
minor raptor tweaks

### PR DESCRIPTION
**Pull request by Ben10Omintrix**
## About The Pull Request
i think i made these a bit too strong, this hopefully balances them out a bit. they were initially accidentally given alot of armor against ranged attacks, this is now removed. also ive reduced their base hp by a bit. im open to other suggestions and feedback

## Why It's Good For The Game
they were stronger than i intended. this keeps their usefulness for battling tendrils, or ore vents but also prevents using them to cheese several megafauna

## Changelog
:cl:
balance: you can now polymorph into raptors
balance: raptors overall have less health and no longer have armor against ranged attacks
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Good change, was going to suggest there's no reason for them to use this subtype.

File: code/modules/mob/living/basic/lavaland/raptor/_raptor.dm [hunk 1]
```diff
@@ -19,16 +19,22 @@ GLOBAL_LIST_EMPTY(raptor_population)
...
 	speed = 2
 	mob_biotypes = MOB_ORGANIC|MOB_BEAST
-	maxHealth = 400
-	health = 400
+	maxHealth = 250
+	health = 250
 	melee_damage_lower = 10
 	melee_damage_upper = 15
+	combat_mode = TRUE
+	status_flags = NONE
```
Comment (Jacquerel) [inline code/modules/mob/living/basic/lavaland/raptor/_raptor.dm:33]: Is it important that they don't have `CANPUSH`?

File: code/modules/mob/living/basic/lavaland/raptor/_raptor.dm [hunk 2]
```diff
@@ -19,16 +19,22 @@ GLOBAL_LIST_EMPTY(raptor_population)
...
+	mob_size = MOB_SIZE_LARGE
+	unsuitable_atmos_damage = 0
```
Comment (Jacquerel) [inline code/modules/mob/living/basic/lavaland/raptor/_raptor.dm:35]: I'm fine with this remaining, but worth checking... Is there a reason for them to be spaceproof?
Can just set them to be able to survive in Lavaland/Icebox but not with 0 pressure or temperature.

Comment (Ben10Omintrix) [reply code/modules/mob/living/basic/lavaland/raptor/_raptor.dm]: good idea

Comment (Ben10Omintrix) [reply code/modules/mob/living/basic/lavaland/raptor/_raptor.dm:34]: actually i dont know if theres a way to make this work off pressure

**Comment by Sadboysuss** [comment]
Maybe it shouldn't be spaceproof cause people used them a lot in space to game, but that's up to you

**Comment by Ben10Omintrix** [comment]
they arent anymore

**Comment by MercuryArrow** [comment]
I do question what appeal they'll have beyond novelty if they lose the lava walking as this will be another hit on the "other lavaland mobs can do this useful thing, but the intended specialist mounts can't".
I think the various colours could use some further specialising and tweaking in terms of armour values and whatnot. 

The real issue right now imo is that yellow raptors aren't really penalised in any way for their speed while retaining the advantages that matter from raptors as a whole.

**Comment by Ben10Omintrix** [comment]
> I do question what appeal they'll have beyond novelty if they lose the lava walking as this will be another hit on the "other lavaland mobs can do this useful thing, but the intended specialist mounts can't".
> 
> I think the various colours could use some further specialising and tweaking in terms of armour values and whatnot. 
> 
> 
> 
> The real issue right now imo is that yellow raptors aren't really penalised in any way for their speed while retaining the advantages that matter from raptors as a whole.

I was thinking I may have some few type of raptors retain their ability to traverse lava. I wasn't aware of how much an issue speedy yellow raptors are, but if so, we can give them some sort of health penalty

**Comment by Ben10Omintrix** [comment]
upon further reflection (unless convinced or forced otherwise), ill leave the lava immunity thing unchanged for now and plan it better for a future pr.

**Comment by LePiromano** [comment]
John Mining seal of approval

Will there be any changes to goliaths considering they're 5 times harder to tame and mount than raptors yet 5 times more useless?

**Pull request merged**